CrossMod/GUI/ModelViewport.cs

```diff
@@ -301,7 +305,7 @@ private void SetUpViewport()
         private static void ClearViewportBuffers()
         {
             GL.Clear(ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit);
-            GL.ClearColor(0.25f, 0.25f, 0.25f, 1);
+            GL.ClearColor(float.Parse(tb1) / 255.0f, float.Parse(tb2) / 255.0f, float.Parse(tb3) / 255.0f, 0);
```

Comment:
The parsing should not be done in the main render loop

Reply:
Ah, just thought of a solution, I can implement it real quick if you eant